ifit/wool
# Closes WEB-12193: Sns handler

Adds Sns Handler subclass, docs, corresponding tests

If you want to test manually, you can:
- clone this locally
- yarn link to a service repo
- deploy din to your aws dev environment
- create an Sns handler function in the service repo
- add necessary serverless config to the test service
- deploy
- trigger SNS message from aws dashboard

or just look through the code and make sure we have test coverage/sufficient documentation
